lib/bolt_ext/server.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'bolt'
+require 'json'
+
+class TransportAPI < Sinatra::Base
+  set :server, :puma
+
+  post '/ssh/run_task' do
+    body = JSON.parse(request.body.read)
+    target = [Bolt::Target.new(body['target']['hostname'], body['target']['options'])]
+    task = Bolt::Task.new(body['task']['name'], body['task']['file'], body['task']['metadata'])
+    parameters = body['parameters']
+
+    @analytics = Bolt::Analytics.build_client
+    executor = Bolt::OrchExecutor.new(Bolt::Config.new, @analytics, false)
+
+    @r = ''
```

Comment:
I don't think we should be setting an empty body. This seems like it will only serve to mask errors.(side note are instance variables on the Base per request?)

We probably need to add some explicit error handling instead.

Reply:
Created https://tickets.puppetlabs.com/browse/BOLT-657